### PR TITLE
[MOBILE-466] enable passive registration

### PR DESCRIPTION
**Pull request by crow**
Enables passive registration and updates Urban Airship SDK version

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,11 @@
 iOS Segment Integration Changelog
 =================================
 
+Version 1.3.0 - Feb 19, 2019
+============================
+- Enable passive registration
+- Update Urban Airship SDK dependency to 10.1.0
```
Comment (BrianBatchelder) [inline CHANGELOG]: Probably should go to` 10.2.0` since it is now out.

**Review by BrianBatchelder — COMMENTED** [review]
One suggestion.

**Review by BrianBatchelder — APPROVED** [review]
🏂

**Comment by rlepinski** [comment]
👍

**Pull request merged**